### PR TITLE
Added generation Plutus.V1.Ledger.Scripts Validator and MP with JSON instances

### PR DESCRIPTION
**Pull request by bladyjoker**
Related to https://github.com/CardaxDEX/blockchain-contracts/issues/293

### CODE REVIEW

File: nix/purescript-bridge-typelib-spago/spago-packages.nix [hunk 1]
```diff
@@ -175,11 +175,11 @@ let
 
     "cardano-transaction-lib" = pkgs.stdenv.mkDerivation {
       name = "cardano-transaction-lib";
-      version = "4604cd9685f085ab12a3055c02188fa2949091ef";
+      version = "db550096e67bfb5ab3ce54134fcdde9cc34f7784";
```
Comment (cstml) [inline nix/purescript-bridge-typelib-spago/spago-packages.nix]: Can we pin this to `staging` or `master` ?

File: nix/purescript-bridge-typelib-spago/packages.dhall [hunk 2]
```diff
@@ -169,11 +170,11 @@ let additions =
           , "uint"
           , "undefined"
           , "unfoldable"
-          , "unsafe-coerce"
           , "untagged-union"
+          , "variant"
           ]
         , repo = "https://github.com/plutonomicon/cardano-transaction-lib.git"
-        , version = "4604cd9685f085ab12a3055c02188fa2949091ef"
+        , version = "db550096e67bfb5ab3ce54134fcdde9cc34f7784"
```
Comment (cstml) [inline nix/purescript-bridge-typelib-spago/packages.dhall]: can we pin this to `staging` or `master`.

**Review by cstml — CHANGES_REQUESTED** [review]

File: nix/purescript-bridge-typelib-spago/spago-packages.nix (hunk 1 above)
Comment (bladyjoker) [reply nix/purescript-bridge-typelib-spago/spago-packages.nix]: For now I need it precisely pinned as I need to check revs across 3 repos (ctl, cardax, pb), when we stabilize the whole setup we'll move to standard Nix pinning as you suggested

File: nix/purescript-bridge-typelib-spago/packages.dhall (hunk 2 above)
Comment (bladyjoker) [reply nix/purescript-bridge-typelib-spago/packages.dhall]: see above

**Comment by bladyjoker** [comment]
Finally managed to get this https://github.com/CardaxDEX/blockchain-contracts/pull/326 working ugh

**Pull request merged**